domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/EmailUtils.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.domain.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.Optional;
+
+public final class EmailUtils {
+
+    private EmailUtils() {
+        // utility class, no instances
+    }
+
+    public static Optional<String> getDefendantEmail(Claim claim) {
```

Comment:
Would be nice to have unit tests for this.

Reply:
oof I was sure I added a test for that. Will do :)